scalding-core/src/main/scala/com/twitter/scalding/bdd/TBddDsl.scala

```diff
@@ -68,7 +68,7 @@ trait TBddDsl extends FieldConversions with TypedPipeOperationsConversions {
 
       val outputPipe = operation(inputPipes) map { Tuple1(_) }
 
-      outputPipe.write(TypedTsv[Tuple1[OutputType]]("output"))
+      outputPipe.write(FixedPathTypedDelimited[Tuple1[OutputType]]("output", "\t"))
```

Comment:
In the process of adding Fields support to the apply methods in TypedSeperatedFile, I made both `TypedSeperatedFile.apply` and `FixedPathTypedDelimited.apply` depend on the macro and use `FixedTypedText` instead of `FixedPathTypedDelimited`. I wanted to do this so that we could eventually get rid of the old `FixedPathTypedDelimited` class, since it seems like `FixedTypedText` and `FixedPathTypedDelimited` perform the same function.

Unfortunately, my change broke this code again by making it depend on the macro. I see two work-arounds:
1) Reverse the changes I just made.
2) Change `FixedPathTypedDelimited` here to `new FixedPathTypedDelimited`, which doesn't require an implicit `TypeDescriptor`. The downside of this approach is that we won't be able to kill the old `FixedPathTypedDelimited` class.
3) Move this class to scalding-commons.

I think the decision comes down to whether we care to keep the old `FixedPathTypedDelimited` class.